.github/workflows/reusable-container-workflow.yaml

```diff
@@ -144,3 +144,15 @@ jobs:
           file: ${{ matrix.dockerfile }}-${{ inputs.build_type }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+      - name: Test Docker run
+        run: |
```

Comment:
mmm, it will happen after the push to repo, so it's too late. can we somehow build it locally and push later. 
or push it as `qa` and then push all the "official" tags?

Reply:
Ah Right! I missed it. We seem to be able to separate build and publish into two steps. Trying that.